packages/@lwc/babel-plugin-component/src/decorators/wire/validate.ts

```diff
@@ -36,13 +41,15 @@ function validateWireParameters(path) {
         });
     }
 
+    // @ts-ignore
     if (isMemberExpression && !id.get('object').isIdentifier()) {
         throw generateError(id, {
             errorInfo: DecoratorErrors.FUNCTION_IDENTIFIER_CANNOT_HAVE_NESTED_MEMBER_EXRESSIONS,
         });
     }
 
     // Ensure wire adapter is imported (check for member expression or identifier)
+    // @ts-ignore
```

Comment:
Ditto

Reply:
This won't be needed if you apply the code change I recommend in the previous comment.

Reply:
I'm trying not to modify the code. Added a TODO.